pkg/system/reconciler.go

```diff
@@ -189,7 +192,8 @@ func NewReconciler(
 	r.ServiceS3.Name = "s3"
 	r.ServiceDb.Name = r.Request.Name + "-db"
 	r.SecretServer.Name = r.Request.Name + "-server"
-	r.SecretDB.Name = r.Request.Name + "-db"
+	r.SecretMongoDB.Name = r.Request.Name + "-db"
+	r.SecretPostgresDB.Name = r.Request.Name + "-db"
```

Comment:
Need to change the names to differentiate between the mongo secret to the postgres